packages/venia-concept/src/RootComponents/Category/category.css

```diff
@@ -19,6 +18,24 @@
     height: 100vh;
 }
 
+.headerButtons {
+    display: flex;
+    justify-content: center;
```

Comment:
Can you add some space between the header items?

[![Image from Gyazo](https://i.gyazo.com/bc29f3b7988147a88814d482c5958e95.png)](https://gyazo.com/bc29f3b7988147a88814d482c5958e95)

Reply:
Added space in latest update